src/pages/TasksDemo.tsx

```diff
@@ -0,0 +1,102 @@
+
+import { Box, Button, Flex, Heading, Input, Stack, HStack, Spinner } from '@chakra-ui/react';
+import { collection, addDoc, updateDoc, deleteDoc, doc, query, CollectionReference, Timestamp, DocumentReference } from 'firebase/firestore';
+import { useState } from 'react';
+import { useCollection } from 'react-firebase-hooks/firestore';
+import { Task } from '../models/Task';
+
+import { db } from '../../firebase.config';
+
+export function TasksPageDemo() {
```

Comment:
It's considered a good practice to use the same naming for both React component and its file
```suggestion
export function TasksDemo() {
```